input/pagecontent/export.md

```diff
@@ -484,7 +484,7 @@ Required Fields:
       <td>An array of deleted file items following the same structure as the <code>output</code> array.
       <br/>
       <br/>
-        The ability to convey deleted resources is important in cases when a server may have previously exported data and wishes to indicate that these data should be removed from downstream systems. When a <code>_since</code> timestamp is supplied in the export request, this array SHOULD be populated with output files containing FHIR Transaction Bundles that indicate which FHIR resources match the kick-off request criteria, but have been deleted subsequent to the <code>_since</code> date. If no resources have been deleted, or the <code>_since</code> parameter was not supplied, or the server has other reasons to avoid exposing these data, the server MAY omit this key or MAY return an empty array. Resources that appear in the 'deleted' section of an export manifest SHALL NOT appear in the 'output' section of the manifest.
+		The ability to convey deleted resources is important in cases when a server may have previously exported data and wishes to indicate that these data should be removed from downstream systems. When a <code>_since</code> timestamp is supplied in the export request, this array SHOULD be populated with output files containing FHIR Transaction Bundles that indicate which FHIR resources would have been returned but have been deleted subsequent to that date. If no resources have been deleted, or if the <code>_since</code> parameter was not supplied, or if the server has other reasons to avoid exposing these data, the server MAY omit this key or MAY return an empty array.		
```

Comment:
```suggestion
The ability to convey deleted resources is important in cases when a server may have previously exported data and wishes to indicate that these data should be removed from downstream systems. When a <code>_since</code> timestamp is supplied in the export request, this array SHOULD be populated with output files containing FHIR Transaction Bundles that indicate which FHIR resources would have been returned but have been deleted subsequent to that date. If no resources have been deleted, or if the <code>_since</code> parameter was not supplied, or if the server has other reasons to avoid exposing these data, the server MAY omit this key or MAY return an empty array.
```